tests/test_utils.py

```diff
@@ -528,6 +529,72 @@ def test_duration(obj, expected_duration):
         assert duration == expected_duration
 
 
+@pytest.mark.parametrize(
+    'index, root, expected',
+    [
+        (
+            audformat.filewise_index(),
+            None,
+            audformat.filewise_index(),
+        ),
+        (
+            audformat.segmented_index(),
+            None,
+            audformat.segmented_index(),
+        ),
+        (
+            audformat.filewise_index(['f1', 'f2']),
+            '.',
+            audformat.filewise_index(
+                [
+                    audeer.safe_path('f1'),
+                    audeer.safe_path('f2'),
+                ]
+            ),
+        ),
+        (
+            audformat.filewise_index(['f1', 'f2']),
+            '.' + os.path.sep,
+            audformat.filewise_index(
+                [
+                    audeer.safe_path('f1'),
+                    audeer.safe_path('f2'),
+                ]
+            ),
+        ),
+        (
+            audformat.filewise_index(['f1', 'f2']),
+            audeer.safe_path(os.path.join('some', 'where')),
+            audformat.filewise_index(
+                [
+                    audeer.safe_path(os.path.join('some', 'where', 'f1')),
+                    audeer.safe_path(os.path.join('some', 'where', 'f2')),
+                ]
+            ),
+        ),
+        (
+            audformat.segmented_index(
+                ['f1', 'f2'],
+                ['1s', '3s'],
+                ['2s', '4s'],
+            ),
+            '.',
+            audformat.segmented_index(
+                [
+                    audeer.safe_path('f1'),
+                    audeer.safe_path('f2'),
+                ],
+                ['1s', '3s'],
+                ['2s', '4s'],
+            ),
+        )
+    ]
+)
+def test_expand_file_path(tmpdir, index, root, expected):
+    expanded_index = audformat.utils.expand_file_path(index, root)
+    pd.testing.assert_index_equal(expanded_index, expected)
```

Comment:
Could you add tests for cases 
* where you have already an absolute path in the index
* where you provide `'some/where/'` as root (I know you have already `./`, but I would also test this with an actual folder).

Reply:
I added a test with absolute path in the index and two more with `'some/where/` and `'some/where`.

Reply:
Cool, thanks.